selenium_tests/test_ui_unit_field.py

```diff
@@ -0,0 +1,36 @@
+# To run these tests, it is required that you perform the following two actions first:
+#   1. Install selenium webdriver and add it to your path variable
+#       https://zwbetz.com/download-chromedriver-binary-and-add-to-your-path-for-automated-functional-testing/
+#   2. Start the server running locally at the URL "http://localhost:8000/"
+
+from selenium import webdriver
+from django.test import LiveServerTestCase, tag
+from selenium.webdriver.common.keys import Keys
+from selenium_tests.selenium_test_base import SeleniumTestBase
+
+
+URL = "http://localhost:8000/"
```

Comment:
Are you using this in your test anywhere or is it just here due to copy and paste?

Reply:
Yeah I just copied from selenium_test_base file. I use the self.driver in the test_ui_unit_field file, so I think I need to import the SeleniumTestBase is that right?

Reply:
Correct. And inherit from it as you are doing